symplyphysics/laws/condensed_matter/resistance_from_temperature.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The resistance depends on the temperature. For different materials, the value
+## of the temperature coefficient and resistance at zero degrees celsius may differ.
+
+## Law is: R = R0 * (1 + a * T), where
+## R - resistance,
+## R0 - resistance at zero degrees celsius,
+## a - temperature coefficient,
+## T - temperature change.
```

Comment:
Change compared to what value?
T is usually used for absolute temperature. Maybe it's better use `dT` or `deltaT` or `T1 - T0`?